src/components/WithinContentContext/index.ts

```diff
@@ -1 +1,4 @@
-export {Provider, Consumer, WithinContentContext} from './WithinContentContext';
+import WithinContentContext from './context';
```

Comment:
This isn't totally strictly related to this PR but i'm not convinced by this component. Acting on this feedback can be done either in this PR or in a new one I don't mind :)

Creating a component that's not really a component, it's just a context feels odd to me. Because it's just a stand alone component it has no uh, context on what it's used for.

What do you think to exposing this from `Banner`? Things that want to set this state like `Card ` can do `import {BannerContentContainer} from 'Banner'. That might help hint that this is used within Banner.

Reply:
I like that approach, it does feel odd having a stand-alone "component" to create the context. I'll add this to the issue